dev_scripts/pytest-wrapper.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# PyTest Wrapper script - temporary solution to tests failing non-deterministically [1]
+# This is only fixed in podman v4.3.0. The wrapper essentially runs the tests in sequence
+# when the podman version is lower.
+#
+# [1]: https://github.com/freedomofpress/dangerzone/issues/217
+
+# FIXME this whole script should be removed and the respective Makefile calling line
+# replaced once all supported platforms have a podman version >= v4.3.0.
+
+import re
+import subprocess
+import sys
+
+import pytest
+from pkg_resources import parse_version
+
+from dangerzone.container import get_runtime_name
+
+PODMAN_MIN_VERSION = "4.3.0"
+
+
+def get_podman_version():
+    result = subprocess.run(
+        "podman version --format '{{.Client.Version}}'", shell=True, capture_output=True
+    )
+    version = result.stdout.decode()[:-1]  # trim trailing \n
+    return version.split("-dev")[0]  # exclude "-dev" suffix from version
+
+
+def run_tests_parallel(pytest_args):
+    args = pytest_args + ["-n", "4"]
+    exit_code = pytest.main(args)
+
+
+def run_tests_sequence(pytest_args):
```

Comment:
Minor, but I think you're looking for "serial" (or "sequentially" as an adverb).

Reply:
Sure, if you prefer it. I have changed it in [495d736](https://github.com/freedomofpress/dangerzone/pull/288/commits/495d73671ab7ea36b4b7f70a1050aa2d1f5f0169).

`run_tests_parallel()` would also need to follow the same logic so I changed it to `run_tests_in_parallel` and `run_tests_in_sequence()`, respectively. Does that sound good?